docs/copied-from-beats/repositories.asciidoc

```diff
@@ -0,0 +1,140 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/setup-repositories.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[setup-repositories]]
+=== Repositories for APT and YUM
```

Comment:
Nice to have this in the docs. Did you actually test that the installation works as expected?

Reply:
I just assumed this would work as we rely on the beats packaging, but we should try it out. I can try to set up a linux machine to try it.

Reply:
Tested on Ubuntnu and CentOS 👍